src/pages/iou/request/step/IOURequestStepScan/index.native.js

```diff
@@ -53,6 +62,7 @@ const defaultProps = {
 
```

Comment:
```suggestion
user: {},
```
in `defaultProps`.  Otherwise `user.isMutedAllSounds` can cause crash before the value `user` is loaded from `Onyx`.